softwareComponents/configuration/include/configuration/rofiworld.hpp

```diff
@@ -662,7 +667,7 @@ class RofiWorld {
         for ( const ModuleInfo& m : _modules ) {
             for ( const ModuleInfo& n : _modules ) {
                 if ( n.module->_id >= m.module->_id ) // Collision is symmetric
-                    break;
+                    continue;
```

Comment:
Ten `break` byl nepřehledný způsob jak to zefektivnit.
Je to ekvivalent k:
```
for i in 1 .. 10:
  for j in 1 .. i:
```

Reply:
Při použití `break` nastává problém, když moduly nejsou seřazeny podle ID.
Např. máme-li v `_modules` moduly 1, 2, 0 , pak s `break` se bude detekovat pouze kolize modulů `m=2` vůči `n=1`; na detekci modulů 0 vůči 1, 0 vůči 2 (či naopak) vůbec nedojde, poněvadž `break` posune vnější čítač `m`:
`break when n >= m`
`m                       1 2 2 0`
`n                       1 1 2 1`
`check collision (n < m) F T F F`
myslím, že continue je v tomhle případě tou správnou optimalizací, pokud nechceme předpokládat seřazenost modulů:
`continue when n >= m`
`m                       1 1 1 2 2 2 0 0 0`
`n                       1 2 0 1 2 0 1 2 0`
`check collision (n < m) F F T T F T F F F`

Reply:
ok, tak to náhraď ekvivalentní podmínkou, která je správně (id1 >= id2 -> continue)

Reply:
Myslím, že tak, jak to je teď, je to správně, či?